management/utils.py

```diff
@@ -179,6 +179,10 @@ def wait_for_service(port, public, env, timeout):
 				return False
 		time.sleep(min(timeout/4, 1))
 
+def get_php_version():
+	# Gets the version of PHP installed in the system.
+	return shell("check_output", ["/usr/bin/php", "-v"])[4:7]
```

Comment:
I don't suppose we'll get to there being a php XX.Y version anytime soon, but slicing the version this way won't work past X.Y version.

Should we instead do something like`/usr/bin/php -r "echo PHP_MAJOR_VERSION . '.' . PHP_MINOR_VERSION;"`?

Reply:
I removed this bit, as I'm reverting to hardcoding the version